components/core/tests/test-StreamingCompression.cpp

```diff
@@ -35,20 +34,14 @@ using std::string_view;
 namespace {
 constexpr string_view cCompressedFilePath{"test_streaming_compressed_file.bin"};
 constexpr size_t cBufferSize{128L * 1024 * 1024};  // 128MB
-// Interleave no-ops to ensure the integrity of the compressor states at all times.
 constexpr auto cCompressionChunkSizes = std::to_array<size_t>(
-        {cBufferSize / 100,
-         0,
+        {0,  // no-op
```

Comment:
Let's remove the command // no-op.
Technically it's not an "no-op", and it is obvious that you are testing a 0 size buffer.